lib/appsignal/hooks.rb

```diff
@@ -65,25 +65,6 @@ def string_or_inspect(string_or_other)
       def truncate(text)
         text.size > 200 ? "#{text[0...197]}..." : text
       end
-
-      def extract_value(object_or_hash, field, default_value = nil, convert_to_s = false)
```

Comment:
Are we sure this is not used by anything else?

Reply:
Yes, I checked before suggesting it :)